x-pack/platform/test/onechat_functional/tests/tools/create_tool.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrProviderContext } from '../../../functional/ftr_provider_context';
+
+const APP_ID = 'agent_builder';
+
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const { common } = getPageObjects(['common']);
+  const testSubjects = getService('testSubjects');
+  const monacoEditor = getService('monacoEditor');
+  const es = getService('es');
+
+  describe('create tool', function () {
+    let testIndexName: string;
+
+    before(async () => {
+      testIndexName = `ftr_onechat_${Date.now()}`;
+      await es.indices.create({ index: testIndexName });
+      await es.index({
+        index: testIndexName,
+        document: { message: 'hello world', numeric: 1, '@timestamp': new Date().toISOString() },
+      });
+      await es.index({
+        index: testIndexName,
+        document: { message: 'second doc', numeric: 2, '@timestamp': new Date().toISOString() },
+      });
+      await es.indices.refresh({ index: testIndexName });
+    });
+
+    after(async () => {
+      try {
+        await es.indices.delete({ index: testIndexName });
+      } catch (e) {
+        // ignore
+      }
+    });
+
+    it('should create an esql tool', async () => {
+      await common.navigateToApp(APP_ID, { path: 'tools/new' });
```

Comment:
you should add the app id to the config `apps` defintion for this `navigateToApp` to work properly, it's likely working now because you're path is `/app/agent_builder` so it's falling back to that when it can't find `agent_builder` in the apps list.

You can define that in you're [config](https://github.com/dennis-tismenko/kibana/blob/agent-builder/tools-ftr-tests/x-pack/platform/test/onechat/common/config.ts#L29) or in the base [config](https://github.com/dennis-tismenko/kibana/blob/agent-builder/tools-ftr-tests/x-pack/platform/test/functional/config.base.ts#L67)